R/utils.R

```diff
@@ -179,7 +179,7 @@ eventTimes <- function(t_events = NULL, t_duration = NULL,
 #' @examples
 #' paramx0 <- createParamList(input_param = rep(99, 10), n_repeat = 10, replace_by_zero = TRUE)
 #' @export
-createParamList <- function(input_param, n_repeat, replace_by_zero = FALSE) {
+.createParamList <- function(input_param, n_repeat, replace_by_zero = FALSE) {
     res_list <- vector(mode = "list", length = n_repeat)
```

Comment:
Seemt so me that this function is not really needed and could be replaced with just 

```
# Initialize result list by replicating the parameters n times
res_list <- lapply(seq_len(n_repeat), function (i) {input_param})
```

This also returns a list where input_param are just repeated n_param times in a list.

-> remove this function and replace it with this basic sequence where appropriate

Reply:
(I can't see here if replace_by_zero has special meaning that can't be easily encoded without a new function)